### ISSUE TITLE
Hitting ctrl+c produced a number of issues and inability to rerun server

### ISSUE THREAD

**Issue by aliak00** [vibe-d/vibe.d#2436]
Starting a vanilla vibe server goes fine until you press ctrl+c

Then I get:

```
Running ./services/api/bin/api 
[main(----) INF] Listening for requests on http://0.0.0.0:8080/
^C[main(----) INF] Received signal 2. Shutting down.
```

And I can't restart the server sometimes as the port is in use and the app is still running in the background. So I have to find the id with `ps` and run `kill -9 id` to be able to rerun the server sometimes

Sometimes this happens:
```
Warning (thread: main): leaking eventcore driver because there are still active handles
   FD 9 (streamListen)
   FD 10 (streamListen)
   FD 13 (streamSocket)
Warning (thread: main): leaking eventcore driver because there are still active handles
   FD 9 (streamListen)
   FD 10 (streamListen)
   FD 13 (streamSocket)
```

And then I can rerun the server.

Related to https://github.com/vibe-d/vibe.d/issues/2245

**Comment by Kozzi11** [comment]
https://github.com/vibe-d/vibe-core/issues/205

**Comment by aliak00** [comment]
Huh. Adding that version really does fix the issue 🤔

**Comment by Geod24** [comment]
The aforementioned issue has been fixed as of eventcore v0.9.11
